css/index.css

```diff
@@ -1,3 +1,456 @@
+}
+
+.boxes > section {
+	background-color: #F4F3F4;
+	border: 1px solid black;
+	width: 48%;
+	padding: 20px;
+	margin-bottom: 40px;
+}
+
+.boxes span {
+	border: 1px solid black;
+	width: 140px;
+	height: 35px;
+	border-radius: 5px;
+	background-color: white;
+	font-size: 80%;
+	display: flex;
+	align-items: center;
+	justify-content: center;
+	margin-top: 20px;
+}
+
+.services-bottom {
+	display: flex;
+	justify-content: space-between;
+}
+
+.services-bottom div {
+	width: 45%;
+}
+
+.contact-header {
+	width: 100%;
+	margin-top: 20px;
+}
+
+.contact-header-text {
+	font-size: 40px;
+	margin-top: 20px;
+	margin-bottom: 10px;
+}
+
+.contact {
+	display: flex;
+	justify-content: space-between;
+}
+
+.contact-content {
+	width: 60%;
+}
+
+.typing {
+	width: 100%;
+	margin-top: 20px;
+}
+
+.contact-content > p {
+	margin-bottom: 10px;
+}
+
+.contact-content ul {
+	margin-top: -10px;
+	margin-left: 5px;
+}
+
+.content-form {
+	width: 40%;
+}
+
+.form {
+	list-style-type: none;
+	list-style-position: inside;
+	line-height: 2;
+	margin-top: 20px;
+	padding: 10px;
+	border: 1px solid black;
+	border-radius: 5px;
+	background-color: #F4F3F4;
+	
+}
+
+.contact-method {
+	list-style-type: none;
+	list-style-position: outside;
+}
+
+.contact-form input {
+	width: 100%;
+}
+
+.contact-form textarea {
+	width: 100%;
+}
+
+button {
+	border: 1px solid black;
+	width: 140px;
+	height: 35px;
+	border-radius: 5px;
+	background-color: white;
+	font-size: 80%;
+	display: flex;
+	align-items: center;
+	justify-content: center;
+	margin-top: 20px;
+	margin-left: 20%;
+	margin-bottom: 10px;
+}
+
+.contact-content {
+	display: flex;
+}
+
+@media (max-width: 800px) {
+
+	.container {
+		width: 775px;
+		padding-left: 10px;
+	}
+
+	header {
+		flex-direction: column-reverse;
+	}
+	
+	nav {
+		padding-top: 35px;
+		padding-left: 10px;
+		width: 775px;
+		justify-content: space-around;
+	}
+
+	.code-image {
+		display: none;
+	}
+
+	.innovation {
+		justify-content: center;
+		width: 775px;
+		padding-left: 0;
+	}
+
+	.middle-img {
+		width: 760px;
+	}
+
+	.middle-content > div {
+		width: 360px;
+	}
+
+	.middle-content {
+		width: 740px;
+		padding-left: 10px;
+	}
+
+	.bottom-content {
+		width: 725px;
+		padding-left: 10px;
+	}
+
+	.bottom-content > div {
+		width: 215px;
+	}
+
+	hr {
+		width: 760px;
+	}
+
+	footer {
+		padding-left: 10px;
+	}
+	
+}
+
+@media (max-width: 500px) {
+	.container {
+		width: 500px;
+		padding-left: 0;
+	}
+
+	hr {
+		width: 460px;
+	}
+
+	header {
+		flex-direction: column-reverse;
+	}
+
+	nav {
+		flex-direction: column;
+		width: 500px;
+		margin-top: -15px;
+	}
+
+	nav a {
+		width: 500px;
+		height: 60px;
+		border-top: 1px solid black;
+		text-align: center;
+		font-size: 20px;
+		padding-top: 20px;
+		padding-left: 50px;
+	}
+
+	nav a:last-child {
+        border-bottom: 1px solid black;
+	}
+	
+	.middle-img {
+		width: 475px;
+		padding-left: 20px;
+	}
+
+	.innovation {
+		width: 500px;
+		margin-top: -25px;
```

Comment:
Try to avoid negative margins - you can take out this negative margin if you lessen the margin on .top :)